packages/mds-agency-cache/index.ts

```diff
@@ -424,7 +434,12 @@ async function writeOneTelemetry(telemetry: Telemetry) {
 }
 
 async function writeTelemetry(telemetries: Telemetry[]) {
-  await Promise.all(telemetries.map(telemetry => writeOneTelemetry(telemetry)))
+  try {
+    await Promise.all(telemetries.map(telemetry => writeOneTelemetry(telemetry)))
+  } catch (err) {
+    logger.error('Failed to write to cache', err)
```

Comment:
nit - no need to identify that it was Telemetry that couldn't be written?

Reply:
Good catch, fixed in f1406ab